sklearn/preprocessing/tests/test_encoders.py

```diff
@@ -0,0 +1,537 @@
+# Authors:
+#
+#          Giorgio Patrini
```

Comment:
It's lonely up here

Reply:
This was just copied from `test_data.py` ... (so for sure he is also not the only author of that :-))
I quickly checked, and I don't think Giorgio contributed to the encoders tests, so is it fine to just remove this alltogether, or do you want me to add myself? (for the the actual implementation I did, not sure if it is needed for the tests)